### PR TITLE
Add "fromValue, velocityX" variables to Swipeable methods + add "onStartDrag" method

### PR DESCRIPTION
**Pull request by lyair1**
I find these few small improvements to Swipeable very useful. Please merge.

### CODE REVIEW

File: react-native-gesture-handler.d.ts [hunk 1]
```diff
@@ -479,14 +479,15 @@ declare module 'react-native-gesture-handler/Swipeable' {
...
     overshootFriction?: number,
-    onSwipeableLeftOpen?: () => void;
-    onSwipeableRightOpen?: () => void;
-    onSwipeableOpen?: () => void;
-    onSwipeableClose?: () => void;
-    onSwipeableLeftWillOpen?: () => void;
-    onSwipeableRightWillOpen?: () => void;
-    onSwipeableWillOpen?: () => void;
-    onSwipeableWillClose?: () => void;
+    onSwipeableLeftOpen?: (fromValue, velocityX) => void;
```
Comment (osdnk) [inline react-native-gesture-handler.d.ts:482]:
```suggestion
    onSwipeableLeftOpen?: (fromValue: string, velocityX: string) => void;
```

File: Swipeable.js [hunk 2]
```diff
@@ -218,28 +222,28 @@ export default class Swipeable extends Component<PropType, StateType> {
     }).start(({ finished }) => {
       if (finished) {
         if (toValue > 0 && this.props.onSwipeableLeftOpen) {
-          this.props.onSwipeableLeftOpen();
+          this.props.onSwipeableLeftOpen(fromValue, velocityX);
```
Comment (osdnk) [inline Swipeable.js:225]: Can we make it an object? 
e.g.
```suggestion
          this.props.onSwipeableLeftOpen({ fromValue, velocityX });
```

I'm not supportive of relying on the order of arguments in such cases. E.g. If I'll need `velocityX` and not `fromValue` then I'll need to write: `onSwipeableLeftOpen = (_, velocityX) => {}`

**Comment by lyair1** [comment]
I Fixed your comments.

**Comment by jakub-gonet** [comment]
@lyair1, could you resolve conflicts and update docs, so it can be merged?

**Comment by jakub-gonet** [comment]
Also, could you remove `onStartDrag` and add some example code in the description? I think we have this implemented in the #999 and it also uses a parameter, so I'm more inclined to merge that one.

**Comment by jakub-gonet** [comment]
@lyair1 🏓

**Comment by jakub-gonet** [comment]
Closing due to no activity. 

We may come back to this PR in the future when redesigning Swipeable API. If you want to continue working on this, please reply. Thanks for your contribution.

**Pull request closed without merging**